.github/workflows/terratests-prod-suite-all.yaml

```diff
@@ -95,8 +96,8 @@ jobs:
           jq --arg timestamp "$TIMESTAMP" '.connection_name = ($timestamp + "_" + .connection_name)' ./tests/examples-without-external-providers/virtual-device-2-azure-connection/terraform.tfvars.json > ./tests/examples-without-external-providers/virtual-device-2-azure-connection/tmp.test.json && mv ./tests/examples-without-external-providers/virtual-device-2-azure-connection/tmp.test.json ./tests/examples-without-external-providers/virtual-device-2-azure-connection/terraform.tfvars.json
           echo $TEST_DATA_PROD_VIRTUAL_DEVICE_2_PORT_CONNECTION >> "./tests/examples-without-external-providers/virtual-device-2-port-connection/terraform.tfvars.json"
           jq --arg timestamp "$TIMESTAMP" '.connection_name = ($timestamp + "_" + .connection_name)' ./tests/examples-without-external-providers/virtual-device-2-port-connection/terraform.tfvars.json > ./tests/examples-without-external-providers/virtual-device-2-port-connection/tmp.test.json && mv ./tests/examples-without-external-providers/virtual-device-2-port-connection/tmp.test.json ./tests/examples-without-external-providers/virtual-device-2-port-connection/terraform.tfvars.json
-          echo $TEST_DATA_PROD_CLOUD_ROUTER_2_VIRTUAL_DEVICE_CONNECTION >> "./examples/cloud-router-2-virtual-device-connection/terraform.tfvars.json"
```

Comment:
This PR is removing the cloud router secrets placement. There shouldn't be any changes to that flow in this PR after the conflict resolution.

Reply:
Addressed with [this](https://github.com/equinix/terraform-equinix-fabric/pull/121/commits/4436554f08ca7f8a5b08904dcd7cd7419e2fd934) commit

Reply:
Removed in the extra file [here](https://github.com/equinix/terraform-equinix-fabric/pull/121/commits/723e9b9cd7a7b9b406a1484e2ecb6e63912e41f8) in the commit above